src/sass/components/cards.scss

```diff
@@ -141,6 +141,15 @@
         &:hover:after {
             @include pill-style-hover(blue);
         }
+
+        &_external-link {
+            @include svg-icon(github-outlined, $size-card-arrow, $size-card-arrow, after);
+
+            &:after {
+                content: '';
```

Comment:
The arrow button in the cards is compounded by a right angle inside a circle. The right angle is a FontAwesome character, and the border is made with a border-box.

The modifier «external_link» on a «cards__element» turns the arrow in the circle button into a github icon.
The github icon should be an outline version according to the designs, but I didn't find it, so I used the filled version removing the button border.
Once the outlined github icon replaces the wrong version it will be according to the design.